apps/app/project.json

```diff
@@ -44,6 +52,12 @@
           ],
           "outputHashing": "all"
         },
+        "en": {
+          "localize": ["en"]
+        },
+        "fr": {
+          "localize": ["fr"]
+        },
         "development": {
```

Comment:
Il ne faudrait pas rajouter un champ `localize` set à `false`, `en` ou `fr` ?
Je me prend une erreur `The development server only supports localizing a single locale per build` sans ça.

Reply:
Yes, je vais changer la configuration par defaut